internal/cmd/base/servers.go

```diff
@@ -223,6 +223,16 @@ func (b *Server) PrintInfo(ui cli.Ui) {
 		b.Info["cgo"] = "enabled"
 	}
 
+	var (
+		tokDurKey = "auth token max duration"
```

Comment:
I'm not sure we want to add these values -- at least not yet. Or maybe we do and will want to wordsmith them later. I'm basically thinking once this is configurable per-auth-method (assuming) this might end up being misleading, especially since we haven't decided the rules on what would override what.

I could go either way, we can leave this in now if you want, but wanted to raise that.

Reply:
I removed it for now because it wasn't totally clear how to plumb through the setting from the configuration file or default to the setting we use in `authtoken.defaultOptions`. I think it would be nice to see it on startup at some point.